tool_collections/samtools/samtools_reheader/samtools_reheader.xml

```diff
@@ -1,5 +1,5 @@
-<tool id="samtools_reheader" name="Replace header" version="1.0.1">
-    <description>on a BAM or SAM file</description>
+<tool id="samtools_reheader" name="Reheader" version="1.2.1">
```

Comment:
I see only name, description, param labels, tests and help changes. And one more format for input_header. 
Is the additional format why it needs a new version?